libarchive/archive_ppmd_private.h

```diff
@@ -72,8 +72,8 @@ typedef struct
 
 typedef struct
```

Comment:
This whole struct should go away as well.

Reply:
thanks, i've trimmed that now too